qiita_db/test/test_util.py

```diff
@@ -750,33 +750,55 @@ def test_generate_biom_and_metadata_release(self):
         tmp = topen(tgz, "r:gz")
         tgz_obs = [ti.name for ti in tmp]
         tmp.close()
-        tgz_exp = [
-            'processed_data/1_study_1001_closed_reference_otu_table.biom',
-            'templates/1_19700101-000000.txt',
-            'templates/1_prep_1_19700101-000000.txt',
-            'processed_data/1_study_1001_closed_reference_otu_table.biom',
-            'templates/1_19700101-000000.txt',
-            'templates/1_prep_1_19700101-000000.txt',
-            'processed_data/1_study_1001_closed_reference_otu_table_'
-            'Silva.biom', 'templates/1_19700101-000000.txt',
-            'templates/1_prep_1_19700101-000000.txt']
-        self.assertEqual(tgz_obs, tgz_exp)
+        # files names might change due to updates and patches so just check
+        # that the prefix exists.
+        fn = 'processed_data/1_study_1001_closed_reference_otu_table.biom'
+        self.assertTrue(fn in tgz_obs)
+        tgz_obs.remove(fn)
+        # yes, this file is there twice
+        self.assertTrue(fn in tgz_obs)
+        tgz_obs.remove(fn)
+        # let's check the next biom
+        fn = ('processed_data/1_study_1001_closed_reference_otu_table_Silva.'
+              'biom')
+        self.assertTrue(fn in tgz_obs)
+        tgz_obs.remove(fn)
+        # now let's check prep info files based on their suffix, just take
+        # the first one and check/rm the occurances of that file
+        fn_prep = [f for f in tgz_obs
+                   if f.startswith('templates/1_prep_1_')][0]
+        # 3 times
+        self.assertTrue(fn_prep in tgz_obs)
```

Comment:
because we created the tests we added more artifacts and didn't assign new file paths so we reused the same ones